lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -22,17 +22,29 @@ defmodule DynamicSupervisor do
 
   The options given in the child specification are documented in `start_link/1`.
 
-  Once the dynamic supervisor is running, we can start children
-  with `start_child/2`, which receives a child specification:
+  Once the dynamic supervisor is running, we can use it to start children
+  on demand. Given this `Counter` GenServer:
```

Comment:
```suggestion
  on demand. Given this sample `GenServer`:
```

let's highlight the GenServer bits.